From 5a1081d36bf2861ffc882354c583a0eb6b0ee3d5 Mon Sep 17 00:00:00 2001
From: Peter Seiderer <ps.report@gmx.net>
Date: Tue, 25 Oct 2016 21:27:41 +0200
Subject: [PATCH] configure.ac: use given CFLAGS/LIBS for mysqlclient library
 check

Needed for static linking of mysql with enabled libz.

Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 configure.ac | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index 5782fd6..aa5d9dd 100644
--- a/configure.ac
+++ b/configure.ac
@@ -618,8 +618,8 @@ else
 		#LDFLAGS="-L$MYSQL_LIBDIR"
 		saved_CFLAGS=$CFLAGS
 		saved_LIBS=$LIBS
-		CFLAGS="-I$MYSQL_INCDIR"
-		LIBS="-L$MYSQL_LIBDIR"
+		CFLAGS="-I$MYSQL_INCDIR $CFLAGS"
+		LIBS="-L$MYSQL_LIBDIR $LIBS"
 		AC_CHECK_LIB(mysqlclient,mysql_init,[
 					TEMP_LIBS="$TEMP_LIBS -L$MYSQL_LIBDIR -lmysqlclient -lz"
 					TEMP_CFLAGS="$TEMP_CFLAGS -I$MYSQL_INCDIR"
-- 
2.8.1

